tests/coverage/coverage.cpp

```diff
@@ -65,9 +65,7 @@ TEST(Coverage, units)
     // Copy constructor
     libcellml::Units uc(um);
 
-    libcellml::Printer printer;
-    const std::string a = printer.printUnits(uc);
-    EXPECT_EQ(e, a);
+    EXPECT_EQ("dimensionless", uc.getName());
```

Comment:
`"dimensionless"` really ought to be declared as a string variable. (Same in `unitsGetVariations